1.1/spec/11-Part-08.adoc

```diff
@@ -492,40 +479,57 @@ The function http://www.opengis.net/def/function/geosparql/asKML[`geof:asKML`] c
+
+NOTE: The two datatypes defined here are for an abstract DGGS implementation (http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`]) and a specific one, AusPIX <<AUSPIX>> (http://www.opengis.net/ont/geosparql#auspixDggsLiteral[`geo:auspixDggsLiteral`]). The purposes of including both of these are to indicate the conceptual position of DGGS literals in this specification - the abstract form acts as a parent of all other potential, specific, DGGS datatypes - and to exemplify a specific system implementation which allows for real examples of use.
 
 ===== RDFS Datatype: geo:dggsLiteral
 
 ```turtle
 geo:dggsLiteral a rdfs:Datatype ;
     rdfs:isDefinedBy geo: ;
-    skos:prefLabel "DGGS Well-Known Text Literal"@en ;
-    skos:definition "A textual serialization of a Discrete Global Grid (DGGS) geometry object."@en .
+    skos:prefLabel "DGGS Literal"@en ;
+    skos:definition "A textual serialization of a Discrete Global Grid System (DGGS) geometry object."@en .
 ```
 
-Valid http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`] instances are formed by encoding geometry information as text and as required by a particular DGGS and in accordance with the _Discrete Global Grid System Abstract Specification_ <<DGGSAS>>. An indication of the particular DGGS, as well as the geometric information must also be indicated in the literal as per the following _ABNF_ <<IETF5234>> syntax specification:
-
-```
-dggsLiteral ::= "<" IRI ">" LWSP geometric-data
-```
+Valid http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`] instances are formed by encoding geometry information according to specific DGGS implmentation. The specific implementation should be indicated by use of a subclass of the `geo:dggsLiteral` datatype. 
 
-The token `IRI` (Internationalized Resource Identifier) is essentially a web address and is defined in <<IETF3987>> and the token `LWSP`, is one or more white space characters, as defined in <<IETF5234>>. `geometric-data` is potentially specific to the DGGS and is not specified here.
+|===
+| *Req 28* All RDFS Literals of type http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`] shall consist of a DGGS geometry serialization formulated according to a specific DGGS.
+|`/req/geometry-extension/dggs-literal`
+|===
 
 |===
-| *Req 28* All RDFS Literals of type http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`] shall consist of a required DGGS identifier, an IRI, and a DGGS geometry serialization formulated according to the identified DGGS.
-|`/req/geometry-extension/dggswkt-literal`
+| *Req 29* An empty RDFS Literal of type http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`], or one of its sud datatypes, shall be interpreted as an empty geometry.
```

Comment:
1. "sud datatypes" should probably  be "subdatatype" (or datasubtype?)
2.  "geometry" should be "Geometry" (the skos:prefLabel) or geo:Geometry in the font used for code.

Reply:
Fixed as requested in commit https://github.com/opengeospatial/ogc-geosparql/pull/162/commits/e8fae58e396d5a1c9762cc55929e3ca0cfa17f9e